README.md

```diff
@@ -0,0 +1,64 @@
+## Perun mobile bindings
+This project provides Android bindings for *go-perun* called *prnm*.  
+Right now it only focusses on two-party-payment channels.  
+
+## Security Disclaimer
+The authors take no responsibility for any loss of digital assets or other damage caused by the use of this software.  
+**Do not use this software with real funds**.
+
+### Getting Started
+```sh
+# Get perun-eth-mobile
+go get -d https://github.com/perun-network/perun-eth-mobile
+# Install gomobile, see https://godoc.org/golang.org/x/mobile/cmd/gomobile
+go get golang.org/x/mobile/cmd/gomobile
+gomobile init
+# Generate the bindings
+gomobile bind -o prnm.aar -target=android github.com/perun-network/perun-eth-mobile/
+```
+
+`prnm.aar` can then be included with Android studio.  
+The easiest way of getting started with an app is to try out the *go-mobile* [example app](https://github.com/golang/go/wiki/Mobile#sdk-applications-and-generating-bindings) and instead of using the `hello.aar` replace it with `prnm.aar` that you generated with the command above.
+
+### In Java
+Go to the `MainActivity.java` of your app and import `prnm.*`. Just to see whether or not the compilation and linking is working, you can try to call `Prnm.contextBackground()` which should return not nil.  
+A sample setup for a two-party-payment channel could look like this:  
+```java
+// You can add a sleep here to ensure that the Android studio logger is attached.
+// Thread.sleep(2000);
+// I hope that this is the correct path for persistent files.
+String appDir = getApplicationContext().getFilesDir().getAbsolutePath();
+String ksPath = appDir +"/keystore";
+String dbPath = appDir +"/database";
+// 10.0.2.2 is the IP of the host PC when using Android Simulator.
+// 8545 is the standart port for ganache-cli.
+String ethUrl = "ws://10.0.2.2:8545";
+// We are alice in this example and this is our on-chain address holding the ETH.
+Address alice = Prnm.newAddressFromHex("0x05e71027e7d3bd6261de7634cf50F0e2142067C4");
```

Comment:
What about importing `Prnm` classes like `Address` and `Config`? Could we add a sample import statement at the top?

Reply:
That all works by `import prnm.*` as i wrote above.